client/oci_env/utils.py

```diff
@@ -64,6 +64,10 @@ def get_config(env_file):
         "DJANGO_SUPERUSER_USERNAME": "admin",
         "DJANGO_SUPERUSER_PASSWORD": "password",
 
+        # Pytest fixtures
+        "AIOHTTP_FIXTURES_ORIGIN": "127.0.0.1",
+        "REMOTE_FIXTURES_ORIGIN": "https://fixtures.pulpproject.org/",
```

Comment:
Too bad, one of these looks like an ip and the other one as a url. But i think the aiohttp port is generated in the fixture, so there is only so much we can do. Maybe, we can hardcode the localhostiness in the fixture eventually.
wdyt @bmbouter ?

Reply:
On the long term the idea is to remove `REMOTE_FIXTURES_ORIGIN` is that right? Is the question here then about what value to use in `AIOHTTP_FIXTURES_ORIGIN` or how to deduplicate them?

Reply:
`AIOHTTP_FIXTURES_ORIGIN` is used by [`gen_threaded_aiohttp_server`](https://github.com/pulp/pulpcore/blob/a710c64244f001a8e947f05fca30342a66104bd5/pulpcore/tests/functional/__init__.py#L394-L398) which provides a facility for creating custom web servers (e.g., [`gen_bad_response_fixture_server`](https://github.com/pulp/pulp_file/blob/6d8a51a2b3fef94015cae9758b15de6dab3a95b5/pulp_file/tests/functional/conftest.py#L309)). `REMOTE_FIXTURES_ORIGIN` is used in tests in all plugins as an entry point for all fixtures. In our CI, we set it to `http://pulp-fixtures:8080`.

It might not be that easy to get rid of `REMOTE_FIXTURES_ORIGIN` right now. Once we will have all the fixtures available as pytest fixtures, then we can think about removing them (a tremendously long-term idea).

Reply:
I think the question was more about what value to use and where.

Reply:
These seem like the right values to me. If it's working then I'd say very much they are the right values.

Reply:
The question is not whether "127.0.0.1" is the right value for our pytest server fixture, but whether we actually need a variable for that value. Can't we just make it a constant in the pulpcore pytest plugin?
Everything configurable about the test suite is driving away from the goal to radically simplify it.

Reply:
I think it could be a constant. The tests are now assumed to be run on the pulp server itself, and the tests coordinate starting up a local server so yes I think this does not need to be configurable.